src/Config/GlobalQuery.ts

```diff
@@ -0,0 +1,32 @@
+import { Query } from '../Query/Query';
+
+export class GlobalQuery {
```

Comment:
Please document via JSDoc the use of this class - which to use `getInstance()` vs `new ...`.

(There is text in `StatusRegistry` or similar that may be useful).

Reply:
done =)